### DISCUSSION TITLE
Upgrade Zabbix LXC

### DISCUSSION THREAD

**Discussion by lilp69** [tteck/Proxmox#4008] [Q&A] [answered]
Hi,
I would know how to upgrade Zabbix when we used the script to install it ?
`apt upgrade grafana -y` don't work.

Thanks.

**Comment by MickLesk** [comment, marked as answer]
Type "update" in LXC?

**Reply by lilp69** [reply]
So easy. Thanks.

**Reply by mikeage** [reply]
I had to also restart the zabbix-server service; `systemctl restart zabbix-server.service`. Without it, I got a new frontend, but the server version was still out of date (from the main dashboard)

**Reply by MickLesk** [reply]
have added this as tweak for the next updates ;)
